.gitignore

```diff
@@ -9,3 +9,4 @@ Thumbs.db
 /.qmake.stash
 
 *.pro.user*
+*.autosave
```

Comment:
You can have it your branch, but we're not gonna merge this as it encourages to forget changes when committing...

Reply:
Reverted.